rds-mysql-eks/sg.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_security_group" "rds" {
+  name        = "rds-${var.sandbox_id}_sg"
+  description = "Allow all inbound traffic"
+  vpc_id      = aws_vpc.main.id
+
+  ingress {
```

Comment:
just a question, not a required change:
is it intentionally this open?
consider rds-mysql for reference:
https://github.com/QualiTorque/terraform/blob/main/rds-mysql/sg.tf#L6